src/app/learn/get-started/basics/build-publish/index.html

```diff
@@ -5,10 +5,10 @@
 
   <stache-page-summary>
     <p>
-      Before you build and publish your SPA, you must first choose a web server to host your site. We recommend <a href="https://pages.github.com/">GitHub Pages</a>, a free tool suitable for front-end projects.
+      Before you build and publish your SPA, you must choose a web server to host your site. We recommend <a href="https://pages.github.com/">GitHub Pages</a>, a free tool suitable for front-end projects.
     </p>
     <sky-alert alertType="info">
-      Blackbaud developers should create repos and provision applications through the SKY UX App Management Portal. Documentation on the SKY UX App Management Portal is available on <a href="https://host.nxt.blackbaud.com/engineering-system-docs/learn/spa/spa-app-portal">the Engineering System website</a>. This site is for Blackbaud internal use only and is not available to external developers.
+      Blackbaud developers create repos and provision applications through the SKY UX App Management Portal. Documentation on the SKY UX App Management Portal is available on <a href="https://host.nxt.blackbaud.com/engineering-system-docs/learn/spa/spa-app-portal">the Engineering System website</a>. This site is for Blackbaud internal use only and is not available to external developers.
```

Comment:
I realize it was already there, can we change that to `https://docs.blackbaud.com/engineering-system-docs/learn/spa/spa-app-portal`?

Reply:
I changed "host" to "docs" in the URL.